compiler-rt/test/asan/TestCases/Windows/allocator_may_return_null_limits.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %clangxx_asan -O0 %s -o %t
+// RUN: %env_asan_opts=allocator_may_return_null=0 not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-ABORT
+// RUN: %env_asan_opts=allocator_may_return_null=1 %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-RETURN_NULL
```

Comment:
(nit: mixing dashes and underscores in the check name doesn't feel right style wise; I think `CHECK-RETURN-NULL` would be more common llvm test style)

Reply:
Thanks, I should have payed more attention (had not realized I was mixing `-` with `_`).
Addressed: https://github.com/llvm/llvm-project/pull/117929/commits/1f66929b1991827bdd068bd5b8f2c1afe72b129e